### ISSUE TITLE
Extra div by using ng-outlet 

### ISSUE THREAD

**Issue by Zizzamia** [angular/angular#5107]
In my page by using `ng-outlet`.
```html
<ng-outlet class="app"></ng-outlet>
```
I aspect something like, where `month-cmp` it's just my directive view.
```html
<ng-outlet class="app" month-cmp>
  <div class="tasks"><div>
</div>
```
Instead I have an extra `div` that will start  creating some possible css conflict.
```html
<ng-outlet class="app ng-scope" month-cmp>
  <div month-cmp>
    <div class="tasks"><div>
  </div>
</div>
```

**Comment by woppa684** [comment]
Also a duplicate of #5078 I guess?

**Comment by pkozlowski-opensource** [comment]
It is indeed duplicate of #5078. Thnx @woppa684 !

**Comment by carlos-** [comment]
@pkozlowski-opensource, I'd have closed the #5078 instead of this bug. Since the description is better here and this one was already tagged by Brian.
